File: app/src/main/java/com/keelim/nandadiagnosis/help/temp/OpenSourceActivity.java
package com.keelim.nandadiagnosis.help.temp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Toast;

import com.google.android.material.snackbar.Snackbar;
import com.keelim.nandadiagnosis.R;

public class OpenSourceActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_open_source);

        Toast.makeText(this, "오픈소스 라이선스 메뉴 입니다.", Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/com/keelim/nandadiagnosis/view/OpenSourceActivity.kt
package com.keelim.nandadiagnosis.view

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.google.android.material.snackbar.Snackbar
import com.keelim.nandadiagnosis.R
import kotlinx.android.synthetic.main.activity_open_source.*

class OpenSourceActivity : AppCompatActivity(R.layout.activity_open_source) {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setSupportActionBar(toolbar)
        toolbar_layout.title = title

    }
}